package.json

```diff
@@ -11,7 +11,7 @@
     "buildCSS": "grunt buildCSS",
     "build": "yarn buildJS && yarn buildCSS",
     "sync": "yarn build && grunt rsync",
-    "develop": "grunt"
+    "develop": "grunt --sourceMap=true"
```

Comment:
Yep we could remove the flag and always have source maps. The reason I put it in however was that I didn't want our CI build generating source maps that'd then be bundled onto the device. However I've taken a look at the iOS/Android build scripts and both exclude maps from their bundle so this isn't actually a problem. 

I'll update so source maps are always generated.